.cache/Keywords/ClickNumber

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please put .cache folder in .gitignore file

Reply:
OK a